\documentclass[11pt]{article}
\usepackage{geometry}
\geometry{
 a4paper,
 total={210mm,297mm},
 left=20mm,
 right=20mm,
 top=20mm,
 bottom=20mm,
 }
 
\usepackage[english]{babel}
\usepackage[utf8x]{inputenc}
\usepackage{amsmath}
\usepackage{graphicx}
\usepackage[colorinlistoftodos]{todonotes}
\usepackage{multicol}
\usepackage{ragged2e}
\usepackage{tocloft}
\usepackage{gensymb}
\usepackage{siunitx}
\usepackage[hypcap]{caption}
\usepackage{capt-of}
\usepackage{setspace}
\usepackage[parfill]{parskip}
\usepackage{amssymb}
\usepackage{textcomp}
\usepackage{float}
\floatstyle{plain}
\usepackage{color}
\usepackage{epstopdf}
\usepackage{natbib}
\usepackage{hyperref}
\hypersetup{
    colorlinks=true,
    linkcolor=blue,
    filecolor=magenta,      
    urlcolor=cyan,
    citecolor=blue
}
\urlstyle{same}
\usepackage{cleveref}

%abstract
\renewenvironment{abstract}
 {\hspace{.8cm}
  {\bfseries\Large\abstractname}
  \list{}{
    \setlength{\leftmargin}{.95cm}%
    \setlength{\rightmargin}{\leftmargin}%
  }%
  \item\relax}
 {\endlist}

%random things needed
%\renewcommand{\cftsecleader}{\cftdotfill{\cftdotsep}} %let the content table have dots to number
\linespread{1.5}  %making one and a half spacing-sh one and half is actually 1.3
\frenchspacing


\begin{document}
\begin{titlepage}
	\centering
	\topskip0pt
	\vspace*{\fill}
	{\huge\bfseries Developing an expression and purification \\ strategy for Cas9 proteins\par}
	\vspace{2cm}
	{\Large Jia Le, Lim {    }  CID: 00865029}
	\\ 	\vspace{0.5cm}
	{Department of Biology, Imperial College London, \\South Kensington Campus, London, U.K.} \\ \vspace{0.5cm}
	{Interned at: Protein Expression and Purification Core Facility, \\European Molecular Biology Laboratory,\\ Heidelberg, Germany} \\
	\vspace*{\fill}
	{\large Supervised by\par
	Dr.~Kim \textsc{Remans} \& Ms Ines \textsc{Racke} }
	\vfill
% Bottom of the page
	{\large Last updated: \today\par}
\end{titlepage}

\vspace*{\fill}
\tableofcontents 
\vspace*{\fill} 
\thispagestyle{empty}

\doublespacing

\newpage
\pagenumbering{gobble}
\vspace*{\fill}
\begin{abstract}  
\doublespacing
The Clustered Regularly Interspaced Short Palindromic Repeats (CRISPR) and CRISPR-associated (Cas) protein system is naturally found in prokaryotes as an adaptive immune response against pathogens. Cas9 proteins interact with double-stranded RNAs (gRNAs) to target and cleave DNA complementary to the protospacer element found on the gRNA. By changing the protospacer element sequence, Cas9 proteins can be easily manipulated for genome engineering, making it increasingly popular in many fields, such as in the study of evolutionary biology or disease modelling. In this research project, a method for expression and purification of Cas9 proteins was tried and tested for its efficiency and ability to produce functional Cas9 proteins. Cas9 proteins were easily purified in high concentrations and were proven effective in \textit{in vitro} assays. Nonetheless, they were not functional in \textit{in vivo} assays, indicating that further modifications are needed for this purification strategy.

\end{abstract}
\vfill


\newpage
\pagenumbering{arabic}
\section{Introduction.} 

\subsection{Genome engineering.}
Genome engineering is a top-down method of synthetic biology, in which the effects of gene modifications are studied. The main aim of genome engineering is to introduce site-specific modifications and to subsequently remove any exogenous DNA or proteins introduced. Currently, much research uses genome engineering, which can correct disease-causing genes, silence viral sequences, uncover gene functions and more~\citep{Gupta2014, Kaminski2016, Ran2013a}.

Zinc finger nucleases (ZFNs) and transcription activator-like effector nucleases (TALENs) are popular approaches of genome engineering. These proteins, when coupled to nuclease domains of restriction enzymes, were able to directly bind DNA and create a double-stranded break (DSB) at specific sites. However, new ZFNs and TALENs have to be designed to target novel DNA sequences as they directly bind to DNA. Design, expression and validation of such proteins are hence difficult and problematic~\citep{Doudna2014}.

Another conventional method involves the use of RNA interference (RNAi). Small RNAs are designed to be complementary to targeted RNA. When introduced into cells, RNAi binds to mRNAs and results in double-stranded RNA that will be degraded by eukaryotic cells. This method reduces the expression of proteins of interest but has been shown to have unpredictable off-target effects and is unable to completely inhibit gene expression. Moreover, RNAi mainly targets mRNA and its inhibition of gene expression is only temporary~\citep{Gilles2014}. Researchers were hence looking for easier and better methods of genome engineering when they came across the Clustered Regularly Interspaced Short Palindromic Repeats (CRISPR) system.

\subsection{Clustered Regularly Interspaced Short Palindromic Repeats (CRISPR).}
The CRISPR system is originally an adaptive immune response used by prokaryotes~\citep{Sorek2013}. Each CRISPR locus comprises of short repeat sequences interrupted by unique spacer sequences derived from viral elements. CRISPR-associated (Cas) proteins bind to CRISPR RNAs (crRNAs) transcribed from CRISPR loci, creating ribonucleoproteins (RNP) that target and cleave viral sequences complementary to crRNA, hence preventing viral infections~\citep{Sternberg2015}. 

There are three main types of CRISPR-Cas systems. Type I and III CRISPR-Cas systems consist of large protein complexes. These systems are hence not suitable for experimental use as multiple proteins have to be introduced into cells for the CRISPR-Cas system to be functional. The Type II system, on the other hand, consists of only one protein, the Cas9 protein and was therefore chosen to develop novel genome engineering methods~\citep{Doudna2014, Jinek2012a}.

\subsubsection{Type II CRISPR-Cas system.}
The DNA endonuclease, Cas9, contains two nuclease domains homologous to HNH and RuvC endonucleases. In this system, the crRNA forms a double-stranded structure with the trans-activating crRNA (tracrRNA). This double-stranded RNA or guide RNA (gRNA) binds to Cas9 proteins and directs them to the targeted DNA sequence, which is base-paired with the protospacer element of the crRNA. Thereafter, Cas9 proteins create a DSB three base pairs upstream of the Protospacer Adjacent Motif (PAM). The PAM is used by prokaryotes to differentiate between exogenous and intracellular DNA and must be present in targeted DNA sequences for DSBs to occur. Single chimeric RNAs were also recently designed to form a similar double-stranded gRNA structure~(\autoref{fig:gRNA.png}). This streamlines the process of gRNA purification or its expression in cells~\citep{Jinek2012a}. 

\begin{figure}[H]
  \centering
    \includegraphics[width=0.95\textwidth]{gRNA.png}
  \captionof{figure}{\footnotesize \doublespacing \textbf{Structure of Cas9.} crRNA binds to tracrRNA to create a double-stranded RNA that guides Cas9 proteins to the cleavage site. Cleavage sites are complementary to the protospacer element sequence (blue bar) found on the crRNA. HNH and RuvC-like domains each cut a single strand of DNA at a site three base pairs upstream of the Protospacer adjacent motif (PAM), creating a double-stranded break. Figure modified from~\cite{Doudna2014}.}
    \label{fig:gRNA.png}
\end{figure}

By changing the 20bp protospacer element of the crRNA~(\autoref{fig:gRNA.png}), researchers can easily manipulate the Cas9 to cleave their genes of interest. This can be performed easily by subcloning the nucleotide sequence into the gRNA backbone plasmid. Moreover, with the introduction of multiple gRNAs, multiple genes can be modified simultaneously with the same protein~\citep{Jinek2012a}. Hence, in contrast to ZFNs and TALENs, Cas9 can be easily designed for genome engineering.

DSBs caused by endonucleases induce homology-directed repair or inaccurate repair by non-homologous end joining (NHEJ). These mechanisms are commonly found in cells and introduce indels at the site of DSB, hence silencing the gene of interest permanently. Until recently, creating DSBs was the rate-limiting step in genome modifications. However, with the introduction of the CRISPR-Cas9 system, this limitation has been largely mitigated~\citep{Chen2015a, Ramalingam2013}. 

\subsection{Cas9 Variants.}
Mutant forms of the wild type \textit{Streptococcus pyogenes} Cas9 (SpyCas9) were subsequently created. Cas9 D10A nickase and Cas9 H840A nickase were created with a non-functional HNH or RuvC endonuclease domain  respectively. Cas9 nickases in combination with paired gRNAs create a DSB when both mutant forms cause nearby single-stranded breaks, hence reducing off-target cleavage~\citep{Ran2013}. 

Another method of reducing unwanted DSBs is to attach Fok1 endonucleases to dead Cas9 (dCas9) proteins, in which both the HNH and RuvC domains are inactivated. When two dCas9 proteins bind to the targeted gene, Fok1 dimerises and creates a DSB. Although dCas9 proteins are non-functional endonucleases, they bind to their targets~\citep{Guilinger2014c}. Hence, transcriptional repressors and activators have been coupled to dCas9 to conduct genome-scale screens for many types of targets, such as tumor suppressor genes and drug resistance genes~\citep{Gilbert2014, Shalem2015}. 

\subsection{Project aim.}
Currently, many researchers transfect their cells with plasmids encoding the gRNA and the Cas9 endonuclease. The drawback of this method is that an incubation period ($\sim$72 hours) is required before mutagenesis effects are detectable. Furthermore, Cas9 will be expressed from the plasmids for several days, which allows the accumulation of off-target mutagenesis effects. Another disadvantage is the possibility of the integration of plasmid DNA into the host cell genome. In contrast, direct transfection with in vitro prepared ribonucleoprotein (RNP) complex leads to a strong peak of activity almost immediately after delivery. The protein gets degraded rapidly afterwards, which reduces chances of off-target effects~\citep{Kim2014}. Therefore, many scientists nowadays prefer to use this approach. 

Nonetheless, this is dependent on the availability of purified Cas9 proteins. Several variants such as the wild type and the nickases are commercially available from vendors such as ThermoFischer Scientific and New England Biolabs, but the prices for the recombinant proteins are rather high. To circumvent these costs and allow scientists to use the direct RNP delivery approach on a larger scale, the EMBL Protein Expression and Purification Core Facility set out to develop a protocol for the in-house production of recombinant Cas9.

In this research, SpyCas9 with differing NLS are expressed, purified and tested for their functionality. By developing a purification strategy, functional Cas9 proteins can be quickly produced for future research. 

\section{Materials and Methods.}\label{sec:methods}

\subsection{Expression and purification of Cas9.}

The first Cas9 fusion protein used in this study carries at the N-terminus a SV40 nuclear localisation sequence (NLS) and a C-terminal His$_{6}$ tag. It will thereafter be known as NLS-Cas9. A second Cas9 fusion protein carries a N-terminal 6xHis-maltose binding protein tag (His$_{6}$-MBP tag), followed by a tobacco etch virus (TEV) protease cleavage site. It also has a C-terminal 2x SV40 NLS, and will thereafter be called Cas9-2NLS~(\autoref{fig:constructs.png}). Two different fusion proteins were used to investigate if the length of NLS affects the efficiency of the protein's translocation into the nucleus.

\begin{figure}[H]
  \centering
    \includegraphics[width=0.8\textwidth]{constructs.png}
  \captionof{figure}{\footnotesize \doublespacing \textbf{(A) NLS-Cas9  fusion protein.} His$_{6}$-tags are used in the purification of fusion proteins. \textbf{(B) Cas9-2NLS fusion protein.}The His$_{6}$-mannose-binding protein (His$_{6}$-MBP) tag of Cas9-2NLS was subsequently removed by the Tobacco Etch Virus (TEV) protease. Both proteins contain the SV40 nuclear localisation sequence (NLS), which is needed for the translocation of the protein into the nucleus.}
    \label{fig:constructs.png}
\end{figure}

NLS-Cas9 and Cas9-2NLS were expressed from pET-NLS-Cas9-6xHis~\citep{Zuris2014} and pMJ915 \citep{Sambrook2001} respectively. Both plasmids were obtained from Addgene. His$_{6}$ tags of both proteins were used as affinity tags for protein purification. The N-terminal His$_{6}$-MBP tag of Cas9-2NLS was eventually cleaved off by TEV protease.

Plasmids were transformed into BL21(DE3) CodonPlus-RIL cells (Stratagene) using standard heat shock protocols with 45 seconds at 42\degree C~\citep{Sambrook2001}. Selection of transformed cells was performed on agar plates containing \SI{100}{\micro\gram}/ml ampicillin and \SI{33}{\micro\gram}/ml chloramphenicol. 

Purification of NLS-Cas9 and Cas9-2NLS were largely carried out as previously described in \cite{Jinek2012a} and \cite{Zuris2014} respectively with some minor modifications. Bacterial cells were grown in Luria-Bertani (LB) medium instead of a 2xTY medium. In lieu of an overnight expression, a short expression of 2NLS-Cas9 was carried out, induced by addition of 0.5mM of $\beta$-D-thiogalactopyranoside(IPTG) at an optical density of A$_{600nm}$ 0.6. After induction, the cells were allowed to grow for an additional 3 hours at 28\degree C. An overnight expression was carried out for NLS-Cas9 as described in \cite{Zuris2014}. Instead of using a homogeniser, all cells were lysed by sonication with 4 cycles of 30 seconds of pulsing at \SI{60}{\micro\metre} amplitude with 30 second pauses. The lysis buffer consisted of 20 mM Tris, 500 mM NaCl and 20mM Imidazole at pH 8.0.

Protein samples were purified by a combination of affinity and ion exchange chromatographic steps. In immobilised-metal affinity chromatography (IMAC), the His$_{6}$ tags bind to the column, allowing Cas9 to be separated from other proteins or from the cleaved His$_{6}$-MBP tag of Cas9-2NLS after digestion by TEV protease. The running buffer for IMAC consisted of 20 mM Tris, 500 mM NaCl, 20mM Imidazole at pH 8.0. The proteins were eluted in a buffer containing 20mM Tris, 250mM NaCl, 10\% glycerol and 300mM Imidazole at pH 8.0. For ion-exchange chromatography (IEX), columns were equilibrated with 20 mM 2-[4-(2-hydroxyethyl)piperazin-1-yl]ethanesulfonic acid (HEPES), 100 mM KCl, 1mM TCEP and 10\% glycerol at pH 7.5 and proteins were eluted in 20 mM HEPES, 1M KCl, 1mM TCEP and 10\% glycerol at pH 7.5. 

Finally, the protein concentration was measured using the Bio-Rad Bradford Protein Assay according to the manufacturer's protocol before being flash-frozen in liquid nitrogen. Both NLS-Cas9 and Cas9-2NLS were divided into 100-250\micro l aliquots and stored in 20 mM HEPES, 300mM KCl, 1mM TCEP and 10\% glycerol at pH 7.5 at -80\degree C.

\SI{1}{\micro\litre} of total cell lysate, \SI{1}{\micro\litre} of cleared lysate (supernatant after centrifugation of total lysate) and \SI{20}{\micro\litre} of protein samples were denatured with loading buffer (0.6\% SDS, 15\% glycerol, 7.5\% $\beta$-mercaptoethanol, 0.0015\% bromphenol blue and 0.075 M Tris HCl, pH 6.8) and incubated at 75\degree C for 10 mins before being ran in sodium dodecyl sulfate polyacrylamide gel electrophoresis (SDS-PAGE). Staining of SDS-PAGEs with Coomassie Brillant Blue was performed according to the standard protocol~\citep{Lawrence2009}.

\subsection{Cas9 Stability and Structural analysis.}
Thermofluor assays were carried out using a CFX Connect$^{TM}$ Real-Time PCR Detection System (Bio-Rad) to analyse the stability of purified Cas9 proteins. Protein samples were stained with SYPRO$^{TM}$ Orange protein gel stain and analysed using the protocol and software from the manufacturer.

One-dimensional nuclear magnetic resonance spectroscopy (1D-NMR) was performed to compare between A5 and A6 NLS-Cas9. The Cas9 storage buffer was exchanged into 20mM Na$_{x}$Po$_{4}$, 100mM NaCl and 0.1mM TCEP at pH 6.9 using a 5mL HiTrap desalting column(GE Healthcare) and concentrated to a concentration of $\sim$80\si{\micro}M using an Amicon ultracentrifugal filter (Millipore, 100-kDa molecular weight cut-off). 

Proteins were analysed as described in~\cite{Anglister1993} and the molecular weights of both proteins are calculated as follows: \\
Molecular Weight (kDa) $\approx 2\tau_{c}$(ns), where  $\tau_{c} \approx \dfrac{1}{5T_{2} (s)}$ at 20\degree C and $T_{2} (s) = \dfrac{2(\Delta_A - \Delta_B)}{\ln(I_A / I_B)}$ \\
$\tau_{c}$, the rotational correlation time, was adjusted to account for the temperature difference using the equation described in~\cite{Daragan1997a}. \textit{I} represents the intensity at time points A and B while $T_{2}$ stands for transverse relaxation time~\citep{Anglister1993}.

\subsection{\textit{In vitro} assays.}
\subsubsection{PCR amplification of target region.}
A 553bp region of the \textit{cdc20} loci, consisting of the target site, was PCR amplified using the \textit{cdc20} forward and reverse primers~(\autoref{table:primers}). The PCR reaction was performed using the Phusion Hot Start II PCR kit (Thermo Fischer Scientific) with 200ng of mouse genomic DNA and the thermocycler at the following setting: one cycle of 98\degree C for 1.5 min, 30 cycles of 98\degree C for 15 s, 64\degree C for 20 s, 72\degree C for 45 s, and one cycle of 72\degree C for 5 min. PCR products were loaded onto a 2\% agarose gel containing SYBR Safe (Life Technologies) in 1xTBE buffer and purified using the QIAquick Gel Extraction Kit (Qiagen). The DNA concentration was measured using NanoDrop (Thermo Fisher Scientific).

\subsubsection{Cas9 RNP assembly and \textit{in vitro} activity assays.} \label{subsubsec:Cas9 RNP assembly}
Alt-R$^{TM}$ CRISPR tracrRNA and crRNA were ordered from IDT with the following protospacer element in the crRNA: 5'-CUCGAACACGAACUGCGCCAGUU-3'. Formation of the crRNA:tracrRNA complex (gRNA) is performed according to the user manual from IDT. 

RNP complexes are assembled right before the experiment with equimolar amounts of gRNA and Cas9 in Cas9 buffer (20mM HEPES, 150mM KCL, 1mM DTT, 5\% glycerol at pH7.5). The components were mixed and left at room temperature for 10 mins. Thereafter, a \SI{20}{\micro\litre} reaction, containing 625nM RNP complex with 104nM PCR product, was incubated in Cas9 buffer with 10mM MgCl$_{2}$ at 37\degree C for one hour. The reaction was quenched by addition of 6x Purple Gel Loading Dye (NEB) and loaded immediately onto a 2\% agarose gel containing SYBR Safe (Life Technologies) in 1xTBE buffer. Commercially produced SpyCas9 from IDT was used as a positive control and the negative control contained no gRNA nor Cas9 proteins.

The experiment was repeated using Cas9 Nuclease Reaction Buffer (NEB) to investigate whether Cas9 proteins are active in the buffer for subsequent \textit{in vivo} assays. The positive control in this experiment comprised of the A6 NLS-Cas9 in Cas9 buffer, shown in the previous experiment to be active.

\label{subsubsec:Cas9 RNP assay analysis}
In order to compare the activities of Cas9 proteins, percentage cleavage was calculated as follows: Percentage cleavage = $\dfrac{b + c }{ a + b + c }$, where a is the band intensity of uncleaved PCR products while b and c reflect those of the cleavage bands. GelQuant.NET software, provided by biochemlabsolutions.com, was used in the analysis of the bands on the agarose gels. 

\subsection{\textit{In vivo} assays.}
HeLa Kyoto cells were cultured in high glucose Dulbecco's modified Eagle's medium (DMEM; Life Technologies) supplemented with 10\% (v/v) fetal bovine serum (FBS), 100 units/ml penicillin, 0.1 mg/ml streptomycin, 2 mM Glutamine and 1 mM (v/v) Sodium pyruvate at 37\degree C and 5\% CO$_{2}$. Cells were passaged the day before to maintain $<$70\% confluency.

\subsubsection{Cas9 RNP assembly and lipofection.}
Alt-R$^{TM}$ CRISPR tracrRNA (IDT) contained the following protospacer element: \\ 5'-ACCAGACGGACACTTACTGA-3', which targeted a region in the \textit{bub1} gene. Formation of the gRNA and RNP complexes were performed similarly as in the \textit{in vitro} assays~(\Cref{subsubsec:Cas9 RNP assembly}). 1:1 and 1:2 ratios of gRNA to Cas9 proteins in 10X Cas9 Nuclease Reaction Buffer (NEB) were topped up to \SI{12.5}{\micro\litre} with Opti-MEM$^{\circledR}$ Medium (Optimem; Thermo Fisher Scientific) in RNP assembly. 

Simultaneously, the transfection complex was formed using a mixture of \SI{1.2}{\micro\litre} of Lipofectamine$^{\circledR}$ RNAiMAX Reagent (Thermo Fisher Scientific) and \SI{11.3}{\micro\litre} of Optimem, which was incubated for 5 mins at room temperature. Thereafter, RNP complexes were added to the transfection complex and incubated at room temperature for 20 mins. Each reaction was subsequently mixed with 10000 cells in a well of a 96-well plate. The experiment was first conducted with 1:1 and 1:2 ratios of 10nM of gRNA to 10nM and 20nM of Cas9 proteins respectively. Afterwards, a 1:1 ratio of gRNA:Cas9-2NLS was used at final RNP concentrations of 10, 60 and 180nM in a subsequent \textit{in vivo} assay.

Cell medium was changed after 4hrs to Optimem with 10\% Fetal Bovine Serium (FBS; Gibco) and incubated at 37\degree C with 5\% CO$_{2}$ for 48 hours to allow NHEJ to take place. Thereafter, the cell medium was removed and \SI{30}{\micro\litre} of lysis buffer (0.45\% Tween, 0.45\% TX100, 2.5mM MgCl$_{2}$, 50mM KCL, 10mM Tris-HCL pH8.3, \SI{100}{\micro\gram/\milli\litre} proteinase K) was mixed with cells. The mixture was left on a rocker for 5 min at room temperature. The extract was then placed in a thermocycler and incubated at 56\degree C for 1 hour, followed by 95\degree C for 10 mins and placed on hold at 4\degree C. 

\subsubsection{PCR amplification of target region.}
The target region was amplified with the \textit{bub1} forward and reverse primers~(\autoref{table:primers}) and the HotStar HiFidelity Polymerase Kit (Qiagen) with \SI{7}{\micro\litre} of the cell extract as template. The thermocycler settings are as follows: 1 cycle of 95\degree C for 5 mins, 40 cycles of 95\degree C for 30s, 60\degree C for 45s, reduced at 0.5\degree C/cycle and maintained at 50\degree C for the last 20 cycles, 72\degree C for 2 mins and 1 cycle of 72\degree C for 2 mins. A nested PCR was used to increase primer specificity and to reduce non-specific amplification.

\subsubsection{T7 endonuclease I assay}
Mismatches caused by the hybridisation of wild-type and mutagenized DNA strands are cleaved by the T7 endonuclease 1 (T7E1), resulting in cleavage bands. The T7E1 assay was performed as described in \cite{Lin2014b} with PCR products. The reaction was quenched with 6x Purple Gel Loading Dye (NEB), resolved on 2\% agarose gel and analysed as described in the \textit{in vitro} assays~(\Cref{subsubsec:Cas9 RNP assay analysis}). Percentage cleavage = (1 -- (1 -- ($\dfrac{b + c}{a + b + c})^{\frac{1}{2}}$)) x 100, where b and c represent the intensity of cleavage bands and a represents the total intensity of uncut PCR products and Cas9 bound PCR products~\citep{Ran2013}. 

\begin{table}[h]
\centering
\caption{\textbf{Primers used in PCR reactions.}}
\label{table:primers}
\begin{tabular}{|l|l|lll}
\cline{1-2}
Primer Names & Primer Sequences\\ \cline{1-2}
\textit{cdc20} forward & 5'-TTGGAACTAGATTTGCCTGGGG-3' \\ \cline{1-2}
\textit{cdc20} reverse & 5'-CGGTTAAGACAGTTGGGTTTTGAA-3' \\ \cline{1-2}
\textit{bub1} forward & 5'-AGAAGAAACGCGACTCTCAG-3' \\ \cline{1-2}
\textit{bub1} reverse & 5'-GGATAATTCGTTAGGCCCTC-3' \\ \cline{1-2}
\end{tabular}
\end{table}

\section{Results.}
\subsection{Purification.}
Both NLS-Cas9 and Cas9-2NLS were produced as soluble proteins in \textit{E.coli}, as can be seen by comparing the total lysate and the cleared lysate samples on the SDS-PAGE~(\autoref{fig:NLS-Cas9_IMAC.png}, \autoref{fig:Cas9-2NLS_1stIMAC.png}). 

However, NLS-Cas9 sample eluted in 2 separate peaks in the ion exchange chromatography (IEX)~(\autoref{fig:NLS-Cas9_IEX.png}). NLS-Cas9 proteins from the first and second peaks were hence stored separately as A5 NLS-Cas9 and A6 NLS-Cas9 respectively.
\\
\\

\begin{figure}[H]
  \centering
    \includegraphics[width=\textwidth]{NLS-Cas9_IMAC.png}
  \captionof{figure}{\doublespacing\footnotesize \textbf{Purification of NLS-Cas9 by immobilised-metal affinity chromatography (IMAC). (A)} NLS-Cas9 was expressed in \textit{E. coli} as a fusion protein containing an N-terminal His$_{6}$ tag. The first purification step consisted of an IMAC. NLS-Cas9 eluted in a single peak, as shown by the absorbances at 280 and 260nm on the chromatogram of the IMAC presented. The IMAC was performed using a 5ml Ni-NTA column (GE Healthcare). \textbf{(B)} SDS-PAGE analysis of IMAC fractions. Analysis was performed on a 4-12\% gradient polyacrylamide gel stained with Coomassie Brillant Blue.}
    \label{fig:NLS-Cas9_IMAC.png}
\end{figure}

\begin{figure}[H]
  \centering
    \includegraphics[width=\textwidth]{NLS-Cas9_IEX.png}
  \captionof{figure}{\doublespacing\footnotesize \textbf{Purification of NLS-Cas9 by ion-exchange chromatography (IEX). (A)} Fractions A2-4 of the IMAC in \autoref{fig:NLS-Cas9_IMAC.png} were further purified by IEX on a 5mL HiTrapSP column (GE Healthcare). NLS-Cas9 eluted in two separate peaks. \textbf{(B)} SDS-PAGE analysis of IEX fractions, carried out as described in \autoref{fig:NLS-Cas9_IMAC.png}. The fractions A5 and A6 were stored separately and will be referred to in  subsequent experiments as A5 NLS-Cas9 and A6 NLS-Cas9 respectively.}
    \label{fig:NLS-Cas9_IEX.png}
\end{figure}

The cleaving of the His$_{6}$-MBP tag of Cas9-2NLS by TEV protease was successful and the cleaved tag was removed by running the sample again over a Ni-NTA column.  Cas9-2NLS no longer binds to the column and was present in the flowthrough as expected~(\autoref{fig:Cas9-2NLS_2ndIMAC.png}). Cas9-2NLS eluted in a single peak in the subsequent IEX~(\autoref{fig:Cas9-2NLS_IEX.png}).

The final concentrations of A5 NLS-Cas9 and A6 NLS-Cas9 were 4.04mg/mL and 9.03mg/mL respectively. The Cas9-2NLS samples had a concentration of 0.8mg/mL.

\begin{figure}[H]
  \centering
    \includegraphics[width=0.80\textwidth]{Cas9-2NLS_1stIMAC.png}
  \captionof{figure}{\doublespacing\footnotesize \textbf{Purification of Cas9-2NLS by immobilised-metal affinity chromatography (IMAC). (A)} Cas9-2NLS was expressed in \textit{E. coli} as a fusion protein containing a N-terminal His$_{6}$-MBP tag. The first purification step consisted of an IMAC. Cas9-2NLS eluted in a single peak, as can be seen in the chromatogram of the IMAC presented here. The IMAC was  performed using a 5ml Ni-NTA column (GE Healthcare). \textbf{(B)} SDS-PAGE analysis of the IEX fractions, performed as described in~\autoref{fig:NLS-Cas9_IMAC.png}.}
    \label{fig:Cas9-2NLS_1stIMAC.png}
\end{figure}

\begin{figure}[H]
  \centering
    \includegraphics[width=0.78\textwidth]{Cas9-2NLS_2ndIMAC.png}
  \captionof{figure}{\doublespacing\footnotesize \textbf{Removal of N-terminal His$_{6}$-MBP tag by IMAC. (A)} After digesting fractions A2 and A3 from the IMAC~(\autoref{fig:Cas9-2NLS_1stIMAC.png}) with TEV protease, the His$_{6}$-MBP tag was successfully removed by its binding to the 5ml Ni-NTA column (GE Heathcare). Cas9-2NLS was found in the flow-through while the last peak in the chromatogram depicted here reflects the tag found in the eluate. \textbf{(B)} SDS-PAGE analysis of IMAC fractions, carried out as described in \autoref{fig:NLS-Cas9_IMAC.png}.}
    \label{fig:Cas9-2NLS_2ndIMAC.png}
\end{figure}

\begin{figure}[H]
  \centering
    \includegraphics[width=\textwidth]{Cas9-2NLS_IEX.png}
  \captionof{figure}{\doublespacing\footnotesize \textbf{Purification of Cas9-2NLS by ion-exchange chromatography (IEX). (A)} Fractions A3 and A4 of the IMAC in \autoref{fig:Cas9-2NLS_2ndIMAC.png} were further purified by IEX on a 5mL HiTrapSP column (GE Healthcare). Cas9-2NLS eluted in a single peak. \textbf{(B)} SDS-PAGE analysis of fractions of IEX, performed as described in \autoref{fig:NLS-Cas9_IMAC.png}. Fractions A4-6 were pooled together and used in subsequent experiments.}
    \label{fig:Cas9-2NLS_IEX.png}
\end{figure}

\subsection{Stability and Structural analysis.}
Using thermofluor assays, the denaturation temperature of both A5 NLS-Cas9 and A6 NLS-Cas9 was determined to be $\sim$44\degree C. This suggests both proteins are probably similarly folded.

The diagram generated by the 1D-NMR confirms the similar folding of A5 NLS-Cas9 and A6 NLS-Cas9~(\autoref{fig:NMR.png}). $\dfrac{I_{A}}{I_{B}}$ = 0.38 for both proteins. As a delay of 1ms was used, $\Delta_A = 1ms$ and $\Delta_B = 0$. After correcting for temperature and using the equations shown in methods~(\Cref{sec:methods}), the molecular weight of both proteins were calculated to be $\sim$165kDa, which agrees well with the actual protein size. This indicates that both A5 NLS-Cas9 and A6 NLS-Cas9 are in a monomeric state.

\begin{figure}[H]
  \centering
    \includegraphics[width=0.8\textwidth]{NMR.png}
  \captionof{figure}{\doublespacing\footnotesize \textbf{1D-NMR shows that A5 NLS-Cas9 and A6 NLS-Cas9 are similar.} Graph plots frequency against intensity of signals obtained in the 1D-NMR experiment performed as in~\cite{Anglister1993}. The diagram shows that the aromatic and amphipathic groups of A5 NLS-Cas9 and A6 NLS-Cas9 are similar as the signals overlap.}
    \label{fig:NMR.png}
\end{figure}

\subsection{Functional analysis.}

All Cas9 proteins were functional in \textit{in vitro} assays. Using the equation listed in methods~(\Cref{sec:methods}), percentage cleavage of PCR products in Cas9 buffer are as follows: 84.0\% for IDT Cas9, 84.0\% for A5 NLS-Cas9, 86.3\% for A6 NLS-Cas9 and 76.0\% for Cas9-2NLS. In NEB buffer, A5 NLS-Cas9, A6 NLS-Cas9 and Cas9-2NLS cleaved 84.6\%, 77.9\% and 54.3\% of PCR products respectively~(\autoref{fig:Invitro.png}). 

\begin{figure}[H]
  \begin{minipage}[c]{0.52\textwidth}
    \includegraphics[width=\textwidth]{Invitro.png}
  \end{minipage}\hfill
  \begin{minipage}[c]{0.45\textwidth}
  \captionof{figure}{\doublespacing\footnotesize \textbf{\textit{In vitro} assay shows that self-produced Cas9 proteins are functional. (A)} 1:1 ratio of gRNA:Cas9 RNP complexes used to cleave PCR products in Cas9 buffer. Cleavage products were resolved on a 2\% agarose gel containing SYBR Safe (Life Technologies) in 1xTBE buffer. \textbf{(B)} 1:1 ratio of gRNA:Cas9 RNP complexes used to cleave PCR products in Cas9 Nuclease Reaction Buffer (NEB).} 
  \label{fig:Invitro.png}
  \end{minipage}
\end{figure}

However, in both \textit{in vivo} assays, self-produced Cas9 proteins were not active, despite the NEB Cas9 in the positive control being functional~(\autoref{fig:cellassay1.png}, \autoref{fig:cellassay2.png}).

\begin{figure}[H]
  \begin{minipage}[c]{0.7\textwidth}
    \includegraphics[width=\textwidth]{cellassay1.png}
  \end{minipage}\hfill
  \begin{minipage}[c]{0.3\textwidth}
  \captionof{figure}{\doublespacing\footnotesize \textbf{Self-produced Cas9 proteins are not functional in cells. } 1:1 and 1:2  ratio of gRNA:Cas9 RNP complexes used to infect HeLa Kyoto cells. T7 endonuclease 1 assay used to detect non-homologous end joining mutations. Cleavage products were resolved on a 2\% agarose gel containing SYBR Safe (Life Technologies) in 1xTBE buffer.} 
  \label{fig:cellassay1.png}
  \end{minipage}
\end{figure}

\begin{figure}[H]
  \centering
    \includegraphics[width=0.8\textwidth]{cellassay2.png}
  \captionof{figure}{\doublespacing\footnotesize \textbf{Self-produced Cas9 proteins are not functional in cells.} 1:1 ratio of gRNA:Cas9 RNP complexes used to infect HeLa Kyoto cells. Analysis carried out as in \autoref{fig:cellassay1.png}.}
    \label{fig:cellassay2.png}
\end{figure}

\newpage
\section{Discussion.}

\subsection{Analysis of self-produced Cas9 proteins.}
Four weeks of expression and purification experiments yielded $\sim$270mg of NLS-Cas9 and $\sim$8mg of Cas9-2NLS. These amounts are sufficient for at least 500 reactions, assuming that all proteins are active and each reaction requires a minimum of 100 pmol of Cas9 proteins~\citep{Lin2014b}. Judging from the SDS-PAGES, all Cas9 proteins are relatively pure and there seem to be little contaminants present~(\autoref{fig:NLS-Cas9_IEX.png}, \autoref{fig:Cas9-2NLS_IEX.png}). Moreover, these proteins are functional in \textit{in vitro} assays~(\autoref{fig:Invitro.png}) and the percentage of PCR products cleaved \textit{in vitro} is comparable to the \% obtained with commercially produced Cas9 and to Cas9 proteins used in other research projects~\citep{Jinek2012a}.

NMR and thermofluor assays~(\autoref{fig:NMR.png}, \autoref{fig:Invitro.png}) show that A5 and A6 NLS-Cas9 have similar denaturation temperatures, are monomeric and process a similar fold. However, it remains unknown why they were eluted in two separate peaks in the IEX~(\autoref{fig:NLS-Cas9_IEX.png}).

The second IMAC of Cas9-2NLS showed a rather jagged chromatogram~(\autoref{fig:Cas9-2NLS_2ndIMAC.png}). A possible explanation might be the fact that the protein was mistakenly dialysed against the IEX running buffer instead of the IMAC running buffer used for equilibration of the respective Ni-NTA column. However, this should not have affected the Cas9-2NLS proteins themselves.

Surprisingly, none of the self-produced Cas9 proteins was functional in the \textit{in vivo} assays~(\autoref{fig:cellassay1.png}, \autoref{fig:cellassay2.png}), although the same buffer and concentration of Cas9 was used both in the \textit{in vitro} and \textit{in vivo} assays. Moreover, a similar approach has been used in the literature~\citep{Zuris2014}. The positive control, which consists of the commercially produced Cas9 protein, was functional in both assays, indicating that there was no major issue with the protocol that was used or with the handling of the cells. As the Cas9 proteins are active in vitro, a possible explanation for this discrepancy might be degradation of the flexible NLS. Although this would not affect the functionality, it would prevent efficient translocation into the nucleus after transfection.

Mass spectrometry analysis can be conducted to show if the NLS is indeed present. This purification strategy can also be further modified by the addition of protease inhibitors to the protein sample during purification, in order to minimise protein degradation. Moreover, Cas9 proteins can also be tagged by fusion to green fluorescent protein (GFP) to show its movement and entry into the cell, to confirm that the problem lies with the translocation of the protein and not with the functionality and/or with the transfection of the cells~\citep{Zuris2014}. Unfortunately, due to time restrictions, these assays could not be carried out anymore during this internship.

\subsection{Protein transfection.}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
As the \textit{in vivo} assays were unsuccessful, it was not possible to compare the efficiency of Cas9 cell delivery methods: protein transfection and plasmid transfection. Nonetheless, current literature suggests that percentage cleavage by Cas9 proteins transfected by both methods are similar~\citep{Kim2014}. However, plasmid transfection requires cells to be incubated for up to $\sim$72 hours as cells have to express Cas9 proteins and its gRNAs before the genome can be edited. In comparison, by directly delivering proteins into cells, cells can be harvested after $\sim$48 hours~\citep{Kleinstiver2016a, Zuris2014}. By shortening the incubation time, the length of the experiment is reduced. Moreover, this has been shown to reduce off-target effects significantly~\citep{Kim2014}.

Furthermore, protein transfection does not involve the introduction of exogenous DNA as the RNP complex is the only substrate introduced into the cell. Cas9 RNP complexes have a half-life of a few hours in cells. Hence, after creating DSBs at specific sites, the complex will be degraded by ribonucleases and proteases. This makes it ideal for human genome editing as it poses no risk of the integration of exogenous DNA into the genome~\citep{Hendel2015, Ramalingam2013}. 

Problems with low efficiency of intracellular delivery of proteins have now been resolved with the use of cationic lipid-mediated delivery~\citep{Zuris2014} and cell-penetrating peptide mediated delivery~\citep{Ramakrishna2014}. With the increased flexibility and lower costs introduced by home-developed purification strategies, more researchers will be able to enjoy the benefits of using direct protein transfection in genome engineering.

\subsection{Limitations of SpyCas9 proteins.}

Nevertheless, many areas of Cas9-mediated gene-editing still require much improvement. 

Off-targeting of Cas9 nucleases remains a problem in gene editing. High fidelity SpyCas9 variants, better gRNA designs and the use of recombinant dCas9 proteins have significantly reduced off-target effects, but more accurate tests, such as genome-wide, unbiased identification of DSBs enabled by sequencing (GUIDE-seq), will be needed to test and reaffirm their off-targeting effects before human trials can begin. As off-targeting effects seem to be dependent on the gRNA sequences, this may limit the number of genes that can be corrected in human cells~\citep{Fu2013a, OGeen2015, Ramalingam2013, Stemmer2015}.

Moreover, in \textit{Streptococcus pyogenes}, the PAM sequence is NGG and this has to be present in the target site for it to be recognised by Cas9. This limits the number of sites that Cas9 can target. However, the range of Cas9 targets can be increased by using Cas9 proteins of other species, which have different PAMs~\citep{Hou2013a}, or by genetic engineering~\citep{Kleinstiver2015}. 

Furthermore, Cas9 proteins have a molecular weight of 160kDa, and are larger than ZFN and TALEN monomers. Since the discovery of SpyCas9, researchers have found smaller Cas9 proteins from other bacteria, such as \textit{Neisseria meningitidis} (\textit{Nme}). The \textit{Nme} Cas9 is 287 amino acids smaller than the one from \textit{Streptococcus pyogenes} and contains a longer PAM sequence. Depending on the gene of interest, the \textit{Nme} CRISPR-Cas9 system can have similar on-target cleavage efficiencies but lower off-target effects, making it more ideal for precise genome engineering. Nonetheless, the \textit{Nme} CRISPR-Cas9 system is not as well-studied as the CRISPR-Cas9 system from \textit{Streptococcus pyogenes}, and will require further trials and modifications to improve its targeting efficiencies~\citep{Lee2016}.

Lastly, the purification strategy developed for SpyCas9 proteins may not necessarily produce other types of functional Cas9 proteins such as the newly discovered Cpf1 endonuclease~\citep{Zetsche2015}. New strategies will have to be developed if such proteins are found to be more efficient in genome engineering.
%%%%%%%%%%%%%%%%%%%%


\section{Conclusion.}
The CRISPR-Cas9 system is a tool that is easy to use and to manipulate in order to achieve specific genetic modifications~\citep{Kaur2015, Sternberg2015}. Thus, it is unsurprising that this system is becoming increasingly popular in research and is currently being used in many different fields such as in the study of evolutionary biology, in the search for novel cures for different diseases and in the genome-scale studies of genes~\citep{Gilbert2014, Kaminski2016, Pal2014}. Due to the growing interest of this system, the gene-editing capabilities of CRISPR-Cas9 and other related endonucleases will hopefully be further improved, thereby boosting the benefits of precise genome engineering.

\newpage 
\vspace*{\fill}
{\huge\bfseries Acknowledgements} \\
\\
\\
\large{I would like to thank Dr.~Kim Remans and Ms Ines Racke at the Protein Expression and Purification Core Facility for taking their time to teach and guide me through this internship. I would also like to thank Ms Birgit Koch and Ms Bianca Nijmeijer for teaching and guiding me through the \textit{in vitro} and \textit{in vivo} assays and Dr. Bernd Simon for conducting the 1D-NMR experiment. Lastly, I would like to thank EMBL for providing the equipment and materials needed for this research.}
\vfill

\newpage
\bibliography{CRISPR2}
\bibliographystyle{cell}


\end{document}

%~(\autoref{fig:locations})
%latex bibtex latex latex ( typeset -> which is command t)
%for tables, type in excel then copy and past to tablesgenerator.com and copy and paste code =D
%pictures use pdf
%texcount /Users/jialelim/Desktop/latex\ annoying\ stuff/TD\ 2015/Ion\ channels\ and\ K+.tex in terminal (command 9)
